Applications/_Plugins/Json/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# =======================================================================
+
+add_library(Plugin_Json INTERFACE)
```

Comment:
This looks like you're using [nlohmann/json](https://github.com/nlohmann/json) which is an external project and should be treated as such. This way if there are any updates to improve the tool that they're easy to integrate.

Reply:
@UebelAndre The files will be moved to common libs and treated as external